config/hg19.clean.yml

```diff
@@ -345,8 +362,108 @@ tracks:
       AN_NFE: an_nfe
       AN_OTH: an_oth
       AN_SAS: an_sas
-    local_files:
-      - gnomad.genomes.r2.0.2.sites.chr*.vcf.bgz
+    remote_files:
```

Comment:
I think we can accept this as is, but in a future PR, lets update annotation sources to expand gnomad fields supported